content/en/real_user_monitoring/data_collected/view.md

```diff
@@ -20,20 +20,35 @@ A page view represents a user visiting a page from your website. During that pag
 
 For page views, loading performance metrics are collected from both the [Paint Timing API][5] and the [Navigation Timing API][6].
 
-For Single Page Applications (SPAs), performance metrics will only be available for the first accessed page. Modern web frameworks like ReactJS, AngularJS, or VueJS update a website’s content without reloading the page, preventing Datadog from collecting traditional performance metrics. RUM is able to track page changes using the [History API][7].
+## Single Page Applications
 
-## Measures collected
+For Single Page Applications (SPAs), the RUM SDK differentiates between `initial_load` and `route_change` navigations with the `loading_type` attribute. If a click on your web page leads to a new page without a full refresh of the page, the RUM SDK will start a new View event with `loading_type:route_change`. RUM is able to track page changes using the [History API][7].
+
+Datadog provides a unique performance metric, Loading Time, which calculates the time needed for a page to load. This metric works both for `initial_load` and `route_change` navigations.
+
+### How is Loading Time calculated?
+To account for modern web applications, Loading Time watches for network requests and long tasks.
+
+* **Initial Load**: Loading Time is equal to the difference between `navigationStart` and:
```

Comment:
```suggestion
* **Initial Load**: loading time is equal to the difference between `navigationStart` and the load event end.
```

Reply:
It's either the load event end or no activity on the page for 100ms